pkg/handler/data_recorder_test.go

```diff
@@ -1,15 +1,39 @@
 package handler
 
 import (
+	"sync"
 	"testing"
 
+	"github.com/checkr/flagr/pkg/config"
+
 	"github.com/prashantv/gostub"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestGetDataRecorder(t *testing.T) {
+	singletonDataRecorderOnce = sync.Once{}
 	defer gostub.StubFunc(&NewKafkaRecorder, nil).Reset()
+
 	assert.NotPanics(t, func() {
 		GetDataRecorder()
 	})
 }
+
+func TestGetDataRecorderWhenKinesis(t *testing.T) {
+	singletonDataRecorderOnce = sync.Once{}
+	defer gostub.StubFunc(&NewKinesisRecorder, nil).Reset()
+	config.Config.RecorderType = "kinesis"
```

Comment:
Let's also set the RecorderType back to default after this test

Reply:
👍